frontend/src/pages/ActivityReport/index.js

```diff
@@ -32,6 +41,9 @@ import {
   reviewReport,
   resetToDraft,
 } from '../../fetchers/activityReports';
+import useLocalStorage from '../../hooks/useLocalStorage';
+import NetworkContext from '../../NetworkContext';
+import { storageAvailable } from '../../hooks/helpers';
```

Comment:
Seems like we are checking if storage is available in a few places, any way we can cut it down to a single place? (if not that is fine!)